hack/lib/install.sh

```diff
@@ -47,6 +47,24 @@ function check_kind {
   fi
 }
 
+# check if golangci-lint installed
+function check_golangci-lint {
+  echo "checking golangci-lint"
+  which golangci-lint >/dev/null 2>&1
```

Comment:
Better to use `command -v`, see https://stackoverflow.com/questions/592620/how-can-i-check-if-a-program-exists-from-a-bash-script